### PR TITLE
[terra-folder-tree] Update docs

### PR DESCRIPTION
**Pull request by sycombs**
### Summary

This PR improves documentation and examples for `terra-folder-tree` so that it is easy for consumers to understand how to consume the component.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [x] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9368 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by sdadn — APPROVED** [review]

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+* **FolderTree**: The top-level folder tree component.
+* **FolderTreeItem**: A row in a folder tree that may have `subfolderItems`.
+  * If there are 1 or more `subfolderItems`, the row will be expandable/collapsible and display a folder icon.
+
+All children of a FolderTree **must** be FolderTreeItems.
+
+### Properties required
+
+The following FolderTree props are required for a basic folder tree:
+* **title**: The title of the folder tree. This is also used as the `aria-label` for the folder tree.
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx:23]: Just an observation, in similar case [here](https://engineering.cerner.com/terra-ui/components/cerner-terra-framework-docs/compact-interactive-list/examples/cell-content#required-properties) the technical writer made the props names to be links, we might want to follow that for consistency.

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import FolderTreeExpandCollapse from './FolderTreeExpandCollapse?dev-site-example';
+
+# Expand/Collapse
+
+### Description
+
+The following example demonstrates expandable/collapsible folders.
+
+### Properties required
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx:9]:
```suggestion
### Required Properties
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Expanded/collapsed state is controlled by the consuming component.
+The following FolderTreeItem props are required for expand/collapse:
+* **isExpanded**: Identifies whether or not the FolderTreeItem is currently expanded.
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx:13]: Same as above, probably should be links to the props table

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import FolderTreeExpandCollapse from './FolderTreeExpandCollapse?dev-site-example';
+
+# Expand/Collapse All
+
+### Description
+
+The following example demonstrates expand/collapse all functionality.
+Expand/collapse all functionality is accessible via the toolbar buttons.
+
+### Properties required
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx:10]:
```suggestion
### Required Properties
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Expanded/collapsed state is controlled by the consuming component.
+The following FolderTree props are required for expand/collapse all:
+* **onExpandAll**: This callback is called when interacting with the Expand All button.
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx:14]: Same as above, probably should be links to the props table

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import BasicFolderTree from './BasicFolderTree?dev-site-example';
+
+# Selection
+
+### Description
+
+The following example demonstrates row selection.
+
+### Properties required
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx:9]:
```suggestion
### Required Properties
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+Selection state is controlled by the consuming component.
+The following FolderTreeItem props are required for row selection:
+* **isSelected**: Identifies whether or not the FolderTreeItem is currently selected.
```
Comment (adoroshk) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx:13]: Same as above, probably should be links to the props table.

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx [hunk 8]
```diff
@@ -30,17 +98,38 @@ This component requires the following peer dependencies be installed in your app
 | react-dom | ^16.8.5 |
 | react-intl | ^2.8.0 |
 
-<!-- AUTO-GENERATED-CONTENT:END -->
+## FolderTree props
```
[the remark this answers is not in the document]
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx]: Updated to "Props Table" here: [3229bd1](https://github.com/cerner/terra-framework/pull/1973/commits/3229bd15cb1231e0372bee951487c4573c55f676)

`FolderTree` and `FolderTreeItem` seem more correct to me since these refer to the specific React components and not just "Folder Tree" or "Folder Tree Item" in general?

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx [hunk 9]
```diff
@@ -1,3 +1,38 @@
...
+### Description
+
+The following example demonstrates a basic folder tree. For more details on specific folder tree
+features, see the following examples:
+* [Selection](./folder-tree-selection)
+* [Expand/Collapse](./folder-tree-expand-collapse)
+* [Expand/Collapse All](./folder-tree-expand-collapse-all)
+
+A folder tree is composed of two components:
+* **FolderTree**: The top-level folder tree component.
```
[the remark this answers is not in the document]
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx]: Updated: [145ae39](https://github.com/cerner/terra-framework/pull/1973/commits/145ae3948663be3dab6cb53c95122c573b286b65)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx [hunk 10]
```diff
@@ -1,3 +1,38 @@
...
+* **FolderTreeItem**: A row in a folder tree that may have `subfolderItems`.
+  * If there are 1 or more `subfolderItems`, the row will be expandable/collapsible and display a folder icon.
+
+All children of a FolderTree **must** be FolderTreeItems.
+
+### Properties required
```
[the remark this answers is not in the document]
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx]: [145ae39](https://github.com/cerner/terra-framework/pull/1973/commits/145ae3948663be3dab6cb53c95122c573b286b65)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx (hunk 4 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx]: [8f2c304](https://github.com/cerner/terra-framework/pull/1973/commits/8f2c304bf1ad74059949d8f8a8101fa0b79a778f)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx (hunk 6 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx]: [8f2c304](https://github.com/cerner/terra-framework/pull/1973/commits/8f2c304bf1ad74059949d8f8a8101fa0b79a778f)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx (hunk 1 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx]: Updated: [0f5b7db](https://github.com/cerner/terra-framework/pull/1973/commits/0f5b7db7c0699ee76faaecbf1800596272a2bb11)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx (hunk 7 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeSelection.2.doc.mdx]: [4b462af](https://github.com/cerner/terra-framework/pull/1973/commits/4b462af48977e5fb036f4ed1a0a9666f2918c9de)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx (hunk 5 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx]: [4b462af](https://github.com/cerner/terra-framework/pull/1973/commits/4b462af48977e5fb036f4ed1a0a9666f2918c9de)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx (hunk 3 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx]: [4b462af](https://github.com/cerner/terra-framework/pull/1973/commits/4b462af48977e5fb036f4ed1a0a9666f2918c9de)

**Review by adoroshk — APPROVED** [review]

**Review by artpark — APPROVED** [review]

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx [hunk 11]
```diff
@@ -30,17 +98,40 @@ This component requires the following peer dependencies be installed in your app
 | react-dom | ^16.8.5 |
 | react-intl | ^2.8.0 |
 
-<!-- AUTO-GENERATED-CONTENT:END -->
+## Properties
+
+### FolderTree Props Table
+<FolderTreePropsTable />
```
Comment (chrismichalewicz) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:104]: I know this table is auto generated but is there way to update the description for the children prop to not include FolderTree.Items?  So it looks like something like "List of items to be displayed as content within the Folder Tree component."

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx [hunk 12]
```diff
@@ -30,17 +98,40 @@ This component requires the following peer dependencies be installed in your app
...
+
+### FolderTreeItem Props Table
+<FolderTreeItemPropsTable />
```
Comment (chrismichalewicz) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:107]: Same comment as above, this time with subfolderItems prop.  If possible replace FolderTree.Items in the Description field with "items" and maybe reword the first sentence to be something like "List of items to display in a subfolder when it is clicked."

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx [hunk 13]
```diff
@@ -2,25 +2,93 @@ import { Notice } from "@cerner/terra-docs";
...
-import FolderTree from './Examples.2/BasicFolderTree?dev-site-example';
+import BasicFolderTree from './Examples.2/BasicFolderTree?dev-site-example';
 
 <Badge />
 
-# Terra FolderTree
+# Terra Folder Tree
 
 <Notice variant="caution" ariaLevel="2">
 Terra Folder Tree is currently under alpha release. There may be breaking changes between releases until it is stable with the 1.0 release.
```
Comment (chrismichalewicz) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:14]: I believe at the completion of this story we should be ready to remove the alpha release designation correct?  If so we will want to remove this notice.

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx (hunk 11 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:104]: Yeah, we can update that from the component. My thought with calling out "FolderTree.Items" specifically is so that consumers know that ONLY FolderTreeItems should be used and not some other React component - do we think that's clear enough if we reword it to just "items"?

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx (hunk 13 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx]: Updated: [b4d6f64](https://github.com/cerner/terra-framework/pull/1973/commits/b4d6f64ca6a5e2347f1febb9a55894d19625dd27)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx (hunk 11 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:100]: Updated: [18cf213](https://github.com/cerner/terra-framework/pull/1973/commits/18cf213eb5eabfc53af77417b4a42ba34b3520f7)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx (hunk 12 above)
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:103]: Updated: [18cf213](https://github.com/cerner/terra-framework/pull/1973/commits/18cf213eb5eabfc53af77417b4a42ba34b3520f7)

**Comment by chrismichalewicz** [comment]
@sycombs  I left a few smaller comments above, please let me know your thoughts on those. In general though, I thought page is well documented, especially the examples. I like how you further explained the individual props needed for each.

I did a quick run through of the examples via keyboard navigation and was able to navigated, expand/collapse, select items as expected.    I also tested them with JAWS on and it all worked as expected and similar to the ARIA site that you linked so good job there. My only question and maybe @eawww  has already addressed this is with the expand/collapse all buttons, once they are selected via keyboard JAWS just says "Enter" there is no confirmation the items were expanded or collapsed.  This shouldn't affect this current PR and am okay with it as is,  I think this is something we can further discuss when Michael does his product validation to determine if we need a follow up story for it.  I just wanted to call it out in the event it had already been discussed somewhere.

**Comment by sycombs** [comment]
> @chrismichalewicz I think that's something I called out on a previous PR or something but probably never logged a jira for it. There should be a state-driven aria-live announcement when all collapsible items are either expanded or collapsed.

UXPLATFORM-10089 was logged to address that!

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx [hunk 14]
```diff
@@ -2,25 +2,93 @@ import { Notice } from "@cerner/terra-docs";
...
+* [Terra Standards](/components/cerner-terra-framework-docs/folder-tree/about#terra-standards)
+* [Properties](/components/cerner-terra-framework-docs/folder-tree/about#properties)
+
+## Overview
+Folder Tree is an accessible [tree view](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/) component that allows the user to display data in a hierarchical tree-like structure of nested rows.
+
+Folder Tree displays an action header containing the title of the folder tree, a toolbar
+with buttons to expand or collapse all items in the folder tree, and zero or more rows which
+may or may not include nested subrows.
+The component contains only one tab stop with the arrow keys used for primary navigation.
```
[the remark this answers is not in the document]
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx]: Updated: [317cdc2](https://github.com/cerner/terra-framework/pull/1973/commits/317cdc22627647559d391a805451323d755e6167)

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+### Required Properties
+
+Expanded/collapsed state is controlled by the consuming component.
+The following FolderTreeItem props are required for expand/collapse:
+* **[isExpanded](../about#foldertreeitem-props-table)**: Identifies whether or not the FolderTreeItem is currently expanded. 
+* **[onToggle](../about#foldertreeitem-props-table)**: This callback is called when interacting with a row that has `subfolderItems` via either of the following methods:
+  * Mouse click:
+    * Clicking anywhere on a row that has `subfolderItems`, other than on the radio button.
+  * Keyboard:
+    * Pressing Right Arrow on a focused, collapsed row that has `subfolderItems`, or Left Arrow on a focused, expanded row that has `subfolderItems`.
```
[the remark this answers is not in the document]
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx:18]: Pressing `Enter` on a folder row selects the row. Should it be expanding/collapsing it instead?

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx (hunk 11 above)
Comment (chrismichalewicz) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx:100]: @sycombs  Thanks for making the update. For where it was being used I think it is clear just stating items as I usually look at field in the table to have a non-technical explanation. Plus you call out FolderTreeItem in the Basic Folder Tree example so I think that is sufficient.

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx [hunk 16]
```diff
@@ -1,3 +1,38 @@
...
+
+The following example demonstrates a basic folder tree. For more details on specific folder tree
+features, see the following examples:
+* [Selection](./folder-tree-selection)
+* [Expand/Collapse](./folder-tree-expand-collapse)
+* [Expand/Collapse All](./folder-tree-expand-collapse-all)
+
+A folder tree is composed of two components:
+* **[FolderTree](../about#foldertree-props-table)**: The top-level folder tree component.
+* **[FolderTreeItem](../about#foldertreeitem-props-table)**: A row in a folder tree that may have `subfolderItems`.
```
[the remark this answers is not in the document]
Comment (sycombs) [reply packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx]: Updated to `FolderTree.Item` since that's how it will be used most of the time. [dd47fb0](https://github.com/cerner/terra-framework/pull/1973/commits/dd47fb0cdd8f5375b8c145e5c19f3ed06b2df5bf)

**Comment by cldeitrich** [comment]
@sycombs, I've pushed my edits to the files, including some edits to the prop tables for consistency. I removed the Features link in the Overview because linking to the subsequent section isn't really necessary. Let me know if any other changes need to be made!

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+
+The consuming component controls the expanded and collapsed state.
+The following FolderTree.Item properties are required for expand and collapse:
+* **[isExpanded](../about#foldertreeitem-props-table):** Identifies whether the FolderTree.Item is currently expanded. 
+* **[onToggle](../about#foldertreeitem-props-table):** This callback is requested when interacting with a row that has `subfolderItems` with one of the following methods:
+  * **Mouse click:**
+    * Selecting anywhere but the radio button on a row that has `subfolderItems`.
+  * **Keyboard:**
+    * Pressing the RIGHT ARROW key on an active, collapsed row that has `subfolderItems`, or the LEFT ARROW key on an active, expanded row that has `subfolderItems`.
+* **[subfolderItems](../about#foldertreeitem-props-table):** Only FolderTreeItems that have `subfolderItems` with one or more items can be expanded and collapsed.
```
Comment (eawww) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx:19]:
```suggestion
* **[subfolderItems](../about#foldertreeitem-props-table):** Only FolderTree.Items that have `subfolderItems` with one or more items can be expanded and collapsed.
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+These FolderTreeItems display a folder icon.
```
Comment (eawww) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx:20]:
```suggestion
These FolderTree.Items display a folder icon.
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+Expand and collapse all functionality is available with the toolbar buttons.
+
+### Required Properties
+
+The consuming component controls the expanded and collapsed state.
+The following FolderTree properties are required for expand and collapse all:
+* **[onExpandAll](../about#foldertree-props-table):** This callback is requested when the user selects the **Expand All** button.
+* **[onCollapseAll](../about#foldertree-props-table):** This callback is requested when the user selects the **Collapse All** button.
+
+Only FolderTreeItems that have `subfolderItems` with one or more items can be expanded and collapsed.
```
Comment (eawww) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx:17]:
```suggestion
Only FolderTree.Items that have `subfolderItems` with one or more items can be expanded and collapsed.
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx [hunk 20]
```diff
@@ -0,0 +1,56 @@
...
+These FolderTreeItems display a folder icon.
```
Comment (eawww) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapseAll.4.doc.mdx:18]:
```suggestion
These FolderTree.Items display a folder icon.
```

File: packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx [hunk 21]
```diff
@@ -1,3 +1,38 @@
...
+* [Selection](./folder-tree-selection)
+* [Expand and Collapse](./folder-tree-expand-collapse)
+* [Expand and Collapse All](./folder-tree-expand-collapse-all)
+
+A folder tree is composed of the following components:
+* **[FolderTree](../about#foldertree-props-table)**: The top-level folder tree component.
+* **[FolderTree.Item](../about#foldertreeitem-props-table)**: A row in a folder tree that may have `subfolderItems`.
+  * If one or more `subfolderItems` exist, a folder icon is displayed for the row, which you can expand and collapse.
+
+All children of a FolderTree must be FolderTreeItems.
```
Comment (eawww) [inline packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx:18]:
```suggestion
All children of a FolderTree must be FolderTree.Items.
```

**Review by eawww — APPROVED** [review]
Found a few more uses of `FolderTreeItem`. Mostly plural so not sure if they should be changed to FolderTree.Items or `FolderTree.Item`s or just left alone.

Approving at any rate. Nice job on this!
